molgenis-apps/src/main/java/org/molgenis/apps/model/App.java

```diff
@@ -86,14 +86,27 @@ public void setIconHref(String iconHref)
 		set(ICON_HREF, iconHref);
 	}
 
+	public Boolean getUseFreemarkerTemplate()
+	{
+		return getBoolean(USE_FREEMARKER_TEMPLATE);
+	}
+
+	public void setUseFreemarkerTemplate(boolean useFreemarkerTemplate)
+	{
+		set(USE_FREEMARKER_TEMPLATE, useFreemarkerTemplate);
+	}
+
 	public FreemarkerTemplate getHtmlTemplate()
 	{
 		return getEntity(LANDING_PAGE_HTML_TEMPLATE, FreemarkerTemplate.class);
 	}
 
 	public void setHtmlTemplate(FreemarkerTemplate htmlTemplate)
 	{
-		set(LANDING_PAGE_HTML_TEMPLATE, htmlTemplate);
+		if (this.getUseFreemarkerTemplate())
```

Comment:
better to remove if if statement, just set the value, the user of the entity can than use the getUseFreemarkerTemplate function to choose to use the template of not.